pages/common/qmmp.md

```diff
@@ -0,0 +1,32 @@
+# qmmp
+
+> An audio player.
+> More information: <http://qmmp.ylsoftware.com>.
+
+- Launch qmmp:
+
+`qmmp`
+
+- Start/stop the currently playing audio:
```

Comment:
```suggestion
- Start or stop the currently playing audio:
```

- This change is optional - no corresponding style guide requirement exists.
- Maybe it's better to use `or` instead of slash. Some maintainers in the past suggested me to do so.